client/wordpress-com.js

```diff
@@ -193,6 +193,16 @@ sections.push( {
 	secondary: true
 } );
 
+if ( config.isEnabled( 'account-recovery' ) ) {
+	sections.push( {
+		name: 'account-recovery',
+		paths: [ '/account-recovery' ],
+		module: 'account-recovery',
+		secondary: false,
+		enableLoggedOut: true,
```

Comment:
I don't think you'll need the pragma unless you've marked the section as `isomorphic: true` (cc @ockham to confirm)